/home1/npiccolotto/ensemble-sets/code/util/geometry.py:282: RuntimeWarning: overflow encountered in long_scalars
  disc = b**2 - 4 * a * c
Traceback (most recent call last):
  File "/home1/npiccolotto/ensemble-sets/code/render2.py", line 585, in <module>
    vis()
  File "/home1/npiccolotto/.pyenv/versions/3.10.13/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home1/npiccolotto/.pyenv/versions/3.10.13/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home1/npiccolotto/.pyenv/versions/3.10.13/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home1/npiccolotto/.pyenv/versions/3.10.13/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home1/npiccolotto/ensemble-sets/code/render2.py", line 581, in vis
    raise e
  File "/home1/npiccolotto/ensemble-sets/code/render2.py", line 543, in vis
    G = ctx.run(render)
  File "/home1/npiccolotto/ensemble-sets/code/render2.py", line 309, in render
    L = route(instance, G, element_set_partition)
  File "/home1/npiccolotto/ensemble-sets/code/util/route.py", line 1360, in route
    route_brosi_ilp(
  File "/home1/npiccolotto/ensemble-sets/code/util/route.py", line 731, in route_brosi_ilp
    x_ew = model.addVars(
  File "src/gurobipy/model.pxi", line 2941, in genexpr
  File "src/gurobipy/tupledict.pxi", line 11, in gurobipy.tupledict.__init__
MemoryError
